tests/unit/test_config.py

```diff
@@ -1805,9 +1805,9 @@ def test_check_task_event_names(item, tmp_flow_config):
                 [[[outputs]]]
                     owl = who
     """)
-    with pytest.raises(WorkflowConfigError) as ex_info:
-        WorkflowConfig('foo', str(flow_file), ValidateOptions())
-    assert str(ex_info.value) == (
+
+    WorkflowConfig('foo', str(flow_file), ValidateOptions())
+    assert log_filter(contains=(
```

Comment:
Need to check the valid events still trigger the handlers

Reply:
I'm not clear what you want me to do here. We are only reading the config here, so I can't see how handlers could be "triggered". That's tested in `tests/f/events`.

Reply:
If you have
```cylc
[runtime]
    [[foo]]
        [[[events]]]
            handler/mail events = submitted, execution timeout, badger, owl
        [[[outputs]]]
            owl = who
```
then we need to make sure the handlers still run for the `submitted`, `execution timeout` and `owl` events. It's not clear that this is the case as what you've done is replaced a raised exception with a logged warning at validation time.